main.go

```diff
@@ -58,8 +59,9 @@ func (c *component) New(ctx context.Context, conf *config) (func(context.Context
 		LogFn:           domain.LoggerFromContext,
 		StatFn:          domain.StatFromContext,
 		AssetAttributor: a,
-		// AttributionFailureHandler field is simply a stub, it varies org to org
+		// AttributionFailureHandler and AttributedAssetValidator fields are stubs, it varies company to company
```

Comment:
nit: I would word this a bit differently, maybe "...are stubs, as the actual implementations of these interfaces will likely vary greatly among different operators of this service."

Reply:
fixed, will change when i push this up